tests/functional/test_wheel.py

```diff
@@ -222,3 +222,18 @@ def test_pip_wheel_with_pep518_build_reqs(script, data):
     assert wheel_file_path in result.files_created, result.stdout
     assert "Successfully built pep518" in result.stdout, result.stdout
     assert "Installing build dependencies" in result.stdout, result.stdout
+
+
+@pytest.mark.network
+def test_pip_wheel_with_pep518_build_reqs_no_isolation(script, data):
+    script.pip('install', 'wheel')
+    script.pip('download', 'setuptools', 'wheel', '-d', data.packages)
```

Comment:
Why are you downloading setuptools and wheel here? Isn't the point of no-isolation that you don't need them?

Reply:
I overlooked it while copy pasting. Whoops! Nice catch! :)